ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlStatementErrorMessage.java

```diff
@@ -24,7 +24,7 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class KsqlStatementErrorMessage extends KsqlErrorMessage {
-  private final String statementText;
+  private final String statementText = "<retracted>";
```

Comment:
Should `statementText` also be removed from constructors?

Reply:
Making this static -- did not want to introduce more changes on the diff by updating the constructors